test/Dialect/FIRRTL/lowerXMR.mlir

```diff
@@ -440,3 +440,31 @@ firrtl.circuit "Top"  {
     // CHECK:  firrtl.strictconnect %memTap_7, %7 : !firrtl.uint<8>
   }
 }
+
+// -----
+
+firrtl.circuit "Top"  {
+  // CHECK-LABEL:  firrtl.module private @DUTModule
+  // CHECK-SAME: in %io_wen: !firrtl.uint<1>, out %io_dataOut: !firrtl.uint<8>)
+  firrtl.module private @DUTModule(in %clock: !firrtl.clock, in %io_addr: !firrtl.uint<3>, in %io_dataIn: !firrtl.uint<8>, in %io_wen: !firrtl.uint<1>, out %io_dataOut: !firrtl.uint<8>, out %_gen_memTap_0: !firrtl.ref<uint<8>>, out %_gen_memTap_1: !firrtl.ref<uint<8>>) attributes {annotations = [{class = "sifive.enterprise.firrtl.MarkDUTAnnotation"}]} {
+    %c1_ui1 = firrtl.constant 1 : !firrtl.uint<1>
+    %rf_memTap, %rf_read, %rf_write = firrtl.mem  Undefined  {depth = 2 : i64, groupID = 1 : ui32, name = "rf", portNames = ["memTap", "read", "write"], readLatency = 0 : i32, writeLatency = 1 : i32} : !firrtl.ref<vector<uint<8>, 2>>, !firrtl.bundle<addr: uint<1>, en: uint<1>, clk: clock, data flip: uint<8>>, !firrtl.bundle<addr: uint<1>, en: uint<1>, clk: clock, data: uint<8>, mask: uint<1>>
```

Comment:
Nit: can the test be golfed down by removing the read and write ports and just leaving the debug port?

Reply:
A memory cannot have only a debug port :P